lib/tests/streamlit/testing/element_tree_test.py

```diff
@@ -39,6 +39,11 @@ def script():
     assert at.success[0].icon == "🚨"
     assert at.warning[0].value == "danger danger"
 
+    repr(at.error[0])
```

Comment:
Are these calls to `repr` to validate that no exception is thrown? Would be useful to add some comments to a few tests (adding the same comment to all of them might be a bit too redundant) so that it's clear what's going on here

Reply:
I added a comment to a few of the test cases, roughly evenly spaced in the file so it'll be hopefully easy to stumble on one if someone's confused.